File: TeamAch.Web/src/app/common/models/project.ts
import { SkillSum } from './skillSum';
import { Skill } from './skill';
import { Member } from './member';

export class Project {
    id: number;
    name: string;
    alias:string;
    color: string;
    skills: Skill[] = [];
    team: Member[] = [];
}

File: TeamAch.Api/TeamAch.Api/Dal/EF/Entities/Project.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TeamAch.Api.Dal.EF.Entities
{
    public class Project
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Alias { get; set; }
        public string Color { get; set; }
        public List<Skill>  Skills { get; set; }
        public List<Member> Team { get; set; }
    }
}